internal/configflags/set.go

```diff
@@ -0,0 +1,92 @@
+package configflags
+
+import (
+	"flag"
+	"net/http"
+	"net/url"
+
+	"github.com/benchttp/runner/config"
+)
+
+// Set reads arguments provided to flagset as config.Fields and binds
+// their value to the appropriate fields of given *config.Global.
+func Set(flagset *flag.FlagSet, cfg *config.Global) {
```

Comment:
```suggestion
// Set reads arguments provided to flagset as config.Fields and binds
// their value to the appropriate fields of given *config.Global.
// The provided *flag.Flagset must not have been parsed yet, otherwise
// bindings its values would fail.
func Set(flagset *flag.FlagSet, dst *config.Global) {
```

I think var name `dst` would convey more meaning about what's going on.

Reply:
Yup!
As for the doc addition, aren't we (if not stating the obvious) trying to explain how Go generally works regarding flags? The change is OK to me in any case.